Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.h

```diff
@@ -129,15 +134,15 @@ class GStreamerMediaEndpoint : public ThreadSafeRefCounted<GStreamerMediaEndpoin
     void prepareDataChannel(GstWebRTCDataChannel*, gboolean isLocal);
     void onDataChannel(GstWebRTCDataChannel*);
 
-    MediaStream& mediaStreamFromRTCStream(String label);
+    MediaStream& mediaStreamFromRTCStream(String mediaStreamId);
```

Comment:
This might be a const String& ?

Reply:
I think I tried indeed, but if you look at the method implementation you'll see the hash takes ownership of the string.